### PR TITLE
Ruby style remarks

### PR DESCRIPTION
**Pull request by filiptepper**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,4 +1,7 @@
 language: ruby
 rvm:
-  - 2.2.3
-before_install: gem install bundler -v 1.10.6
+  - 2.5.1
+  - 2.4.4
+  - 2.3.7
+  - 2.2.10
```
Comment (nijikon) [inline .travis.yml:6]: We should drop 2.2; upstream support has ended.

**Review by nijikon — DISMISSED** [review]

File: .travis.yml (hunk 1 above)
Comment (filiptepper) [reply .travis.yml:6]: I don't plan on supporting 2.2, it's just a check to see if it's still compatible for people still on 2.2.

**Comment by filiptepper** [comment]
@nijikon @mensfeld When reviewing please keep in mind that we're not using this gem anymore and we don't really want to invest too much time in maintaining it.

**Comment by nijikon** [comment]
@filiptepper let's just merge this as is

**Review by nijikon — APPROVED** [review]

**Pull request merged**